Examples/restcomm-olympus-nopod/restcomm-olympus/MainTableViewController.mm

```diff
@@ -732,15 +732,8 @@ - (UIButton *)getBugReportButton{
     return bugButton;
 }
 
-
-- (void)rcRegisterPushSuccessfully{
-   
-    NSLog(@"Push Register success");
+-(void)rcRegisteredForPush:(NSError *)error{
```

Comment:
Let's avoid the prefix rc, as we haven't used it in any other RCConnection/RCDevice callback. Remember this is public API and we strive for consistency ;)

Maybe just registeredForPush or pushDidRegister?

Reply:
registeredForPush it is.